src/test/java/org/minions/devfund/carlosorellana/KataCountCharactersTest.java

```diff
@@ -0,0 +1,46 @@
+package org.minions.devfund.carlosorellana;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * The goal is test the count characters methods.
+ *
+ * @author Carlos Orellana.
+ * @since 07/06/2018.
+ */
+public class KataCountCharactersTest {
```

Comment:
Your implementation is failing with the following unit test
```java
        int numberCharacters = this.kataCountCharacters.countCharacteres("fizzbuzz", 'Z');
        Assert.assertEquals(4, numberCharacters);
```

Reply:
I modify my code to works correctly